testcases/PowerNVDump.py

```diff
@@ -1069,6 +1069,95 @@ def runTest(self):
             else:
                 log.info("PASSED: Hugepage size is {} kB".format(hugepage_size))
 
+class OpTestMakedump(PowerNVDump):
+    '''
+    function will test the vmcore is captured or not and run the makedumpfile
+    '''
+
+    def kernel_crash(self):
+        '''
+        This function will test the kdump followed by system
+        reboot. it has below steps
+        1. Take backup of files under /var/crash.
+        2. Trigger kernel crash: ``echo c > /proc/sysrq-trigger``
+        '''
+        self.c.run_command("mkdir -p /var/crash_bkp/")
+        if len(self.c.run_command("ls -1 /var/crash")) != 0:
+            self.c.run_command("mv /var/crash/* /var/crash_bkp/")
+            self.c.run_command("sync")
+            time.sleep(5)
+        else:
+            log.info("/var/crash/ directory is empty")
+        self.c.run_command("echo 1 > /proc/sys/kernel/sysrq")
+        try:
+            self.c.run_command("echo c > /proc/sysrq-trigger")
+        except (KernelOOPS, KernelKdump):
+            self.cv_SYSTEM.goto_state(OpSystemState.OS)
+
```

Comment:
Now that we are inheriting PowerNVDump to OpTestMakeDump can we use kernel_crash of PowerNVDump class. Just to avoid duplication and confusion in future.